src/Microsoft.Health.Api/Registration/ApplicationBuilderExtensions.cs

```diff
@@ -21,10 +21,12 @@ public static class ApplicationBuilderExtensions
         /// </summary>
         /// <param name="app">Application builder instance.</param>
         /// <param name="healthCheckPathString">Health check path string.</param>
-        public static void UseHealthChecksExtension(this IApplicationBuilder app, string healthCheckPathString)
+        /// <param name="predicate">A predicate that is used to filter the set of health checks executed.</param>
+        public static void UseHealthChecksExtension(this IApplicationBuilder app, string healthCheckPathString, Func<HealthCheckRegistration, bool> predicate)
```

Comment:
yes, it maybe, we will also change the code in fhir server, or add a default predicate to avoid breaking, which one you think is better? or any other suggestions?

Reply:
Yes can we add a default value here?

Reply:
add null as default